### PR TITLE
174-fix-select-for-darkly

### PR DESCRIPTION
**Pull request by OyvindLGjesdal**

### CODE REVIEW

File: bootstrap-themes/darkly/bootswatch.less [hunk 1]
```diff
@@ -129,7 +131,8 @@ table,
 // Forms ======================================================================
 
 input,
-textarea {
+textarea,
+select {
```
Comment (OyvindLGjesdal) [inline bootstrap-themes/darkly/bootswatch.less:135]: Is select used outside of forms somewhere else, should it maybe be more specified?

**Comment by NateWr** [comment]
Thanks @OyvindLGjesdal! Can you update this to only add the `select` selector? I'd like to minimize the changes for a stable version.

Also, can you base your PR against the `3_2_0_4` branch? That's the one I can release against. The current `master` branch is not compatible with OJS 3.3.x.

**Comment by NateWr** [comment]
Sorry, I meant to say can you base your PR against the `stable-3_3_0` branch.

**Comment by OyvindLGjesdal** [comment]
Thank you for reviewing! I now only add the select selector and have changed the base branch to stable_3_3_0.

**Pull request merged**